src/modules/flow/iio/iio.json

```diff
@@ -421,8 +421,8 @@
            "name": "use_device_default_scale"
          },
          {
-             "data_type": "float",
-             "default": 0,
+             "data_type": "direction-vector",
+             "default": {},
```

Comment:
@barbieri I didn't get the 'afterwards' part. This option is actually to define in which scale(or offset) device is operating...
Anyway, as `sol_rgb` doesn't hold doubles, using three separated options should be the way to go. Using `direction-vector` is misleading...